### PR TITLE
update for SoapFault public variables

### PR DESCRIPTION
**Pull request by Artistan**

### CODE REVIEW

**Comment by andrey-sokolov** [comment]
Thank you for contribution. Please, add some link with documentation for this fields. Please, add PHP version info

File: memcache/memcache.php [hunk 1]
```diff
@@ -384,7 +384,13 @@ public function pconnect ($host, $port, $timeout = 1) {}
  */
 function memcache_connect ($host, $port, $timeout = 1) {}
 
-function memcache_pconnect () {}
```
Comment (Artistan) [inline memcache/memcache.php]: http://php.net/manual/en/memcache.pconnect.php

File: soap/soap.php [hunk 2]
```diff
@@ -640,6 +640,31 @@ public function addSoapHeader (SoapHeader $object) {}
  * @link http://php.net/manual/en/class.soapfault.php
  */
 class SoapFault extends Exception  {
```
Comment (Artistan) [inline soap/soap.php]: http://php.net/manual/en/soapfault.soapfault.php

Comment (Artistan) [reply soap/soap.php]: (PHP 5 >= 5.0.1, PHP 7)

File: memcache/memcache.php (hunk 1 above)
Comment (Artistan) [reply memcache/memcache.php]: (PECL memcache >= 0.4.0)

**Pull request merged**